csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.Nuget.cs

```diff
@@ -428,17 +432,18 @@ private bool CheckFeeds(List<FileInfo> allFiles)
             var feeds = GetAllFeeds(allFiles);
 
             var excludedFeeds = Environment.GetEnvironmentVariable(EnvironmentVariableNames.ExcludedNugetFeedsFromResponsivenessCheck)
-                ?.Split(Path.PathSeparator, StringSplitOptions.RemoveEmptyEntries)
+                ?.Split(" ", StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Maybe add an example as a part of the environment variable constant and describe that whitespace is used as separator. Also, I think if you do `.Split()` then whitespace (probably including `\t`) will also be considered a delimiter.